### PR TITLE
gnome-extra/gnome-software: Version bump to 43.0

### PR DESCRIPTION
**Pull request by Schievel1**
Signed-off-by: Pascal Jäger <pascal.jaeger@leimstift.de>

### CODE REVIEW

File: gnome-extra/gnome-software/gnome-software-43.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+inherit gnome.org gnome2-utils meson virtualx xdg
+
+DESCRIPTION="Gnome install & update software"
+HOMEPAGE="https://wiki.gnome.org/Apps/Software https://gitlab.gnome.org/GNOME/gnome-software"
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flatpak +firmware gnome gtk-doc sysprof udev +webapps hardcoded-foss-webapps hardcoded-proprietary-webapps"
```
Comment (mattst88) [inline gnome-extra/gnome-software/gnome-software-43.0.ebuild:14]: Flags in `IUSE` should be sorted alphabetically.

File: gnome-extra/gnome-software/gnome-software-43.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		$(meson_use gtk-doc gtk_doc)
+		-Dhardcoded_curated=true
+		-Ddefault_featured_apps=false # TODO: Will this be beneficial to us with flatpak at least? If enabled, it shows some apps under installed (probably merely due to /usr/share/app-info presence), but launching and removal of them is broken
+		-Dmogwai=false #TODO?
+		$(meson_feature sysprof)
+		-Dprofile=''
+		-Dsoup2=true
+		$(meson_use webapps)
+		$(meson_use hardcoded-proprietary-webapps hardcoded_proprietary_webapps)
+		$(meson_use hardcoded-foss-webapps hardcoded_foss_webapps)
```
Comment (mattst88) [inline gnome-extra/gnome-software/gnome-software-43.0.ebuild:90]: These three new options should go between `rpm_ostree` and `gudev`.

Secondly, should we actually even have the new `USE` flags? If not for `glib-testing` (which we'll have to figure out), it doesn't looks like they add any dependencies.

I'd say let's just enable them by default and not add the `USE` flags if we don't have to.

**Review by mattst88 — CHANGES_REQUESTED** [review]

File: gnome-extra/gnome-software/gnome-software-43.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+
+LICENSE="GPL-2+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="flatpak +firmware gnome gtk-doc sysprof udev +webapps hardcoded-foss-webapps hardcoded-proprietary-webapps"
+REQUIRED_USE="
+	hardcoded-foss-webapps? ( webapps )
+	hardcoded-proprietary-webapps? ( webapps )
+"
+RESTRICT="test" # See TODO below, TODO needs glib-testing-0, which I can not find.
```
Comment (mattst88) [inline gnome-extra/gnome-software/gnome-software-43.0.ebuild:19]: Strange. Maybe it's https://gitlab.gnome.org/pwithnall/libglib-testing ?

Comment (Schievel1) [reply gnome-extra/gnome-software/gnome-software-43.0.ebuild:19]: Will try, thanks. I feel like I looked though half of the whole internet yesterday and couldn't find it

**Comment by listout** [comment]
Duplicate of #27287

**Comment by Schievel1** [comment]
duplicate of #27287

**Pull request closed without merging**